src/Mono.Android/Xamarin.Android.Net/X509TrustManagerWithValidationCallback.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+
+using Android.Runtime;
+
+using Javax.Net.Ssl;
+
+using JavaCertificateException = Java.Security.Cert.CertificateException;
+using JavaX509Certificate = Java.Security.Cert.X509Certificate;
+
+namespace Xamarin.Android.Net
+{
+    [Register ("xamarin/android/net/X509TrustManagerWithValidationCallback")]
```

Comment:
This repo uses tabs for indentation, not spaces.  Thank you.

Reply:
What's odd is that [`.editorconfig`](https://github.com/xamarin/xamarin-android/blob/main/.editorconfig) *should* be formatting this file appropriately.  What editor are you using?  Does it support `.editorconfig` files?

Reply:
I'm using VS Code so it should pick up the config just fine 🤔 I'll convert the spaces to tabs, no problem. Thanks for pointing it out!